index.html

```diff
@@ -103,6 +103,11 @@
       }
     };
     */
+    /*
+    Nuxeo.UI.config.resource = {
+      uncancelable: true,
+    };
+    */
```

Comment:
By default, shouldn't we actually have this uncommented? Otherwise, when serving with `index.html` we will `uncancelable` set to `false`, which is different when what we will get wit the `index.jsp`, right?
```suggestion
    Nuxeo.UI.config.resource = {
      uncancelable: true,
    };
```

Reply:
You are totally right, let me update it.